src/app/organizations/collection/collection.component.html

```diff
@@ -63,12 +67,8 @@ <h4>
               </mat-card-subtitle>
             </mat-card-header>
 
-            <mat-card-content class="p-3">
-              <mat-chip-list>
-                <mat-chip *ngFor="let label of entry?.labels">{{ label.value }}</mat-chip>
-              </mat-chip-list>
-            </mat-card-content>
             <mat-card-actions *ngIf="(canEdit$ | async)">
+              <div fxFlex></div>
```

Comment:
Not what it's intended for, creates a 0 height element.  Meant to actually have content inside it.  Just do the same thing as add-entry.component.html